third_party/2and3/redis/client.pyi

```diff
@@ -117,7 +117,7 @@ class Redis(object):
     def delete(self, *names): ...
     def __delitem__(self, name): ...
     def dump(self, name): ...
-    def exists(self, name): ...
+    def exists(self, names: Iterable[Text]) -> int: ...
```

Comment:
Looks like this is actually `*names: Text`

Reply:
Thanks 😄